x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/pre_check.ts

```diff
@@ -47,3 +45,16 @@ const checkVmRunner = async () => {
     );
   }
 };
+
+const checkSsh = async () => {
+  const { log } = getRuntimeServices();
+
+  try {
+    const version = await execa.command('ssh -V');
+
+    log.verbose(`Using 'ssh': ${version.stdout} ${version.stderr}`);
```

Comment:
🤦  for whatever reason, the output goes to `stderr` although the command exists with `0`